Story/StoryHooks.cs

```diff
@@ -89,9 +91,26 @@ private void StoryHooks()
             On.Menu.SlugcatSelectMenu.SliderSetValue += SlugcatSelectMenu_SliderSetValue;
             On.Menu.SlugcatSelectMenu.SetChecked += SlugcatSelectMenu_SetChecked;
             On.Menu.SlugcatSelectMenu.GetChecked += SlugcatSelectMenu_GetChecked;
+            On.Menu.PauseMenu.SpawnExitContinueButtons += PauseMenu_SpawnExitContinueButtons;
         }
 
 
+
+        private void PauseMenu_SpawnExitContinueButtons(On.Menu.PauseMenu.orig_SpawnExitContinueButtons orig, Menu.PauseMenu self)
+        {
+            orig(self);
+            if (isStoryMode(out var story) && showRestartButton)
+            {
+                var restartCycle = new SimplerButton(self, self.pages[0], self.Translate("Restart"), new Vector2(self.exitButton.pos.x - (self.continueButton.pos.x - self.exitButton.pos.x) - self.moveLeft - self.manager.rainWorld.options.SafeScreenOffset.x, Mathf.Max(self.manager.rainWorld.options.SafeScreenOffset.y, 15f)), new Vector2(110f, 30f));
+                restartCycle.OnClick += (_) =>
+                {
+                    showRestartButton = false;
```

Comment:
can't rely on it getting unset from clicking, we'll have to unset it at game start or something like that

Reply:
... that won't get unset if you're revived though

imo fixing that is way too much hassle, thoughts on always adding the restart button for host? then we can get rid of `showRestartButton`

Reply:
You don't have to twist my arm.